app/build.gradle

```diff
@@ -77,6 +77,7 @@ dependencies {
 
     // Material Design
     implementation "androidx.compose.material:material:1.2.1"
+    implementation 'androidx.compose.material3:material3:1.0.0-alpha11'
```

Comment:
@DSteve595 This is what was included with a new Compose project, but it's an alpha version. Thoughts?

Reply:
Beta 1 is out now